common/cb_types.txt

```diff
@@ -735,7 +783,7 @@ annex_core_country = {
 
 	badboy_factor = 0
 	prestige_factor = 5
-	peace_cost_factor = 100
+	peace_cost_factor = 80
```

Comment:
What's the reasoning here? The ai will accept any peacedeal when it hits 100 war score (assuming it doesn't go beyond like 103 - 104 warscore)

Reply:
The declaring AI always peaces out and lessening the peace_cost_factor prevents it. Maybe it's because the peace deal shows up as 101 war score, IDK, but I figured out how to fix this old bug. BTW, it's only 60 in other mods (I haven't played Vanilla in years).

Reply:
I've never seen an AI issue with it when they've occupied everything -- if there are unoccupied places and the warscore isn't at 100, I don't think that a realm should be accepting annexation. You got some screenshots that support a realm being fully occupied, but the ai won't accept the peacedeal?

Reply:
I sent the screenshots in #general in the mod's server. Gondor just randomly decided to peace out with Dol Amroth despite being on the verge of completely occupying it and having a lot of war score. My change fixed it.

Reply:
Hmmm. This feels like a poor solution to a complex problem. Was Gondor at 50% battle war score?

Reply:
Gondor was at almost 100. 
Same happened when Umbar tried to Establish Protectorate on Harnendor after eating most of it and occupying it's remaining 2 provinces. 
When Gondor tried to liberate south Gondor from the Harnendan rump state, it white peaces because of the cost.
100 war score seems to make AI do this (maybe because it shows up as 101 in the peace deal).

BTW, in other mods annexation CBs are always less than 100. Conquest is 85, for example.

Reply:
Let's put it at 85 then -- I imagine that other mods have done much more thorough testing

Reply:
Other mods have 85 for conquest and 60 for Restore Order.

Reply:
We can always lower it later. Let's try 85 for now and see if that is low enough to work with the AI